### PR TITLE
Added profile context

### PR DESCRIPTION
**Pull request by hendri-tobing**
Added profile context and profile creation when registering.

### CODE REVIEW

File: lib/vutuv/accounts/user.ex [hunk 1]
```diff
@@ -12,6 +12,8 @@ defmodule Vutuv.Accounts.User do
           reset_sent_at: DateTime.t() | nil,
           posts: %Ecto.Association.NotLoaded{} | [Post.t()],
           sessions: %Ecto.Association.NotLoaded{} | [Session.t()],
+          email_addresses: %Ecto.Association.NotLoaded{} | [EmailAddress.t()],
+          profile: %Ecto.Association.NotLoaded{} | [Profile.t()],
```
Comment (riverrun) [inline lib/vutuv/accounts/user.ex]: The type should be `Profile.t()` (not a list) if there is just one profile.

File: lib/vutuv/biographies/profile.ex [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    |> validate_length(:honorific_suffix, max: 80)
+    |> validate_length(:headline, max: 255)
+  end
+
+  defp validate_first_name_or_last_name(changeset, %{}) do
+    first_name = get_field(changeset, :first_name)
+    last_name = get_field(changeset, :last_name)
+
+    if first_name || last_name do
+      # No error if any of those 2 are present.
```
Comment (riverrun) [inline lib/vutuv/biographies/profile.ex]: Again, let's remove the comments

**Review by riverrun — CHANGES_REQUESTED** [review]
Just a couple of minor changes requested

**Comment by hendri-tobing** [comment]
> Just a couple of minor changes requested

These are done. Thank you.

**Review by riverrun — APPROVED** [review]

**Pull request merged**